examples/csvs/orders.csv

```diff
@@ -0,0 +1,2 @@
+Name,Processed At,Currency,Email,Note,                                         Billing First Name,Billing Last Name,Billing Address1,Billing Address2,Billing City,Billing Company,Billing Zip,Billing Phone,Billing Country Code,Billing Province Code,Shipping First Name,Shipping Last Name,Shipping Address1,Shipping Address2,Shipping City,Shipping Company,Shipping Zip,Shipping Phone,Shipping Country Code,Shipping Province Code,Lineitem sku,Lineitem quantity,Lineitem price,Shipping Line Title,Lineitem fulfillment status,Shipping Line Price,Financial Status
```

Comment:
Not sure it has any implication in reading the .csv file but there's a huge number of spaces after `Note,` here.

Reply:
We strip the file headers before doing the Import. But I will delete those spaces anyway!